docs/segment.md

```diff
@@ -2,98 +2,188 @@
+
+- Correction
+- Main
+- Numbering
+- Signature
+
+You can use them, or create your own. To deactivate default line types, you can deactivate them by unticking the ones you want to discard, and then by clicking on the update button.
+
+You can then create your own by writing the text region name you want into the dedicated field and then the **+** button, and then save your modifications with the **upload** button.
+
+Then, once you are back to the segmentation panel, you can add a label to the text line you drew earlier by clicking on them and then by clicking on the 'Set the type on all selected lines/regions' button, or by pressing ++t++. A dropdown menu should appear with the custom labels you created.
 
 !!! warning
-    It is advisable to ensure the quality of the segmentation before changing the order of the lines because adding or deleting lines systematically restarts the calculation of this order, overwriting manual modifications in the process.
\ No newline at end of file
+    If you created text regions to annotate text lines, you will usually want your text lines to be linked to them. To do so, select your baselines, and click on the **link selected lines to the background region**, or press ++y++. If you want to unlink them, just press the button again, or press ++u++.
+
+!!! note
+    It is possible to automatically perform [text lines segmentation](predict.md#predict-the-segmentation) with a machine learning model. 
+
+### Reordering text lines
+
+The reading order of the text lines is calculated automatically. You can display the order number of each text lines from the “Segmentation” panel by clicking on the **Toggle ordering display** button or by pressing ++l++.
+
+You can also go to the “Text” panel (also accessible with the shortcut ++ctrl+5++) where the lines are displayed in the reading order.
+
+In this panel, and after the text lines are transcribed, it is possible to modify the reading order by clicking “Toggle sorting mode.” A simple drag and drop is enough to carry out the modifications.
+
+![image: Line reordering](img/segment/line_order.gif "You can reorder text lines in eScriptorium.")
+
+!!! warning
+    Please make sure the quality of the segmentation is good enough before changing the order of the lines because adding or deleting lines restarts the line ordering, overwriting manual modifications in the process.
+
+### Advanced text lines segmentation features
+
+- You can add coordinate points on a text line by double-clicking  at the desired location on the baseline.
+
+![image: Adding new coordinates points to a baseline.](img/segment/adding_point_baseline.gif "You can add coordinate points on a text line by double-clicking  at the desired location on the baseline.")
+
+- You can move a coordinate point with a simple click and drag.
+
+![image: Moving coordinates point on a baseline.](img/segment/move_point_baseline.gif "You can move a coordinate point with a simple click and drag.")
```

Comment:
Shouldn't it be "coordinate points" with the S on the second word rather than the first one or maybe on both